assets/css/index.css

```
@@ -265,28 +309,37 @@ img {
 }
-
 .itemType {
-	font-style: italic;
 	margin-top: 5px;
+	position: absolute;
+	padding: 5px;
+	background: #1310108a;
+	color: white;
+	left: 5px;
+	bottom: 5px;
 	margin: top 8px;
```

Stylelint (17.14.0)

[error] Overridden property "margin-top" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)